pkg/virt-handler/vm.go

```diff
@@ -34,6 +34,8 @@ import (
 	"sync"
 	"time"
 
+	"kubevirt.io/kubevirt/pkg/network/driver"
```

Comment:
can we import this as "networkdriver" or "netdriver" since the context is "virt-handler/vm" it will give the code the missing "network" context.

Reply:
You are definitely right. Will do.